src/main/recorder.ts

```diff
@@ -279,19 +266,9 @@ const glob = require('glob');
     /**
      * Reconfigure the underlying obsRecorder. 
      */
-    reconfigure = (
-        outputPath: string, 
-        maxStorage: number, 
-        monitorIndex: number, 
-        audioInputDeviceId: string, 
-        audioOutputDeviceId: string, 
-        minEncounterDuration: number
-    ) => {
-        this._maxStorage = maxStorage;
-        this._minEncounterDuration = minEncounterDuration;
-
+    reconfigure = (options: RecorderOptionsType) => {
```

Comment:
Don't we need to set the _options here? I think we're going to end up in a state where we have reconfigured the obsRecorder but not the member variables on this class. We might just get away with it but definitely confusing.

Reply:
Dang it, you're correct! This is also why it would be a bad idea to expand the options into separate member variables due to having to do it twice.